external/js/cothority/package-lock.json

```diff
@@ -4836,13 +4836,13 @@
       "dependencies": {
         "ansi-regex": {
           "version": "3.0.0",
-          "resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-3.0.0.tgz",
```

Comment:
I don't understand why all of these resolved lines were changed. Here is a bug that mentions it might be due to using npm link, which i know you use during development. https://npm.community/t/npm-install-or-npm-update-turns-a-bunch-of-resolved-in-package-lock-json-from-real-values-to-false/3308

Reply:
Ok so I cleaned my npm cache and stuff and it turns out to fix the resolved. It still removes some optional fields but those are less worrying.